templates/order.json

```diff
@@ -0,0 +1,185 @@
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-event",
+    "compaction": {
+      "keep_versions": 0,
+      "time_threshold_hours": 168
+    },
+    "namespaced_identifiers": false,
+    "source": {
+      "type": "http_endpoint"
+    },
+    "transform": [
+      {
+        "rules": {
+          "default": [
+            [
+              "copy",
+              "*"
+            ],
+            [
+              "add",
+              "_id",
+              [
+                "string",
+                "_S.id"
+              ]
+            ]
+          ]
+        },
+        "type": "dtl"
+      }
+    ],
+    "type": "pipe"
+  },
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-event-processed",
+    "namespaced_identifiers": false,
+    "source": {
+      "dataset": "{{@ system @}}-{{@ datatype @}}-event",
+      "type": "dataset"
+    },
+    "transform": {
+      "rules": {
+        "default": [
+          [
+            "if",
+            [
+              "eq",
+              "_S.event",
+              "{{@ datatype @}}.delete"
+            ],
+            [
+              "add",
+              "_deleted",
+              true
+            ]
+          ],
+          [
+            "merge",
+            "_S.value"
+          ]
+        ]
+      },
+      "type": "dtl"
+    },
+    "type": "pipe"
+  },
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "namespaced_identifiers": false,
+    "source": {
+      "datasets": [
+        "{{@ system @}}-{{@ datatype @}}-all",
+        "{{@ system @}}-{{@ datatype @}}-event-processed"
+      ],
+      "type": "merge_datasets"
+    },
+    "type": "pipe"
+  },
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-all",
+    "type": "pipe",
+    "source": {
+      "type": "embedded",
+      "entities": [{
+        "start": "~t2023-01-01T00:00:00Z"
+      }]
+    },
+    "transform": [{
+      "type": "dtl",
+      "rules": {
+        "default": [
+          ["add", "_id",
+            ["datetime-format", "%Y-%m-%d", "_S.start"]
+          ],
+          ["add", "::properties",
+            ["dict", "type", "{{@ datatype @}}", "query",
+              ["concat", "{{@ datatype @}}DateFrom=",
+                ["datetime-format", "%Y-%m-%d", "_S.start"], "&{{@ datatype @}}DateTo=",
+                ["datetime-format", "%Y-%m-%d",
+                  ["datetime-plus", "month", 2, "_S.start"]
+                ]
+              ]
+            ]
+          ],
+          ["add", "::operation", "get-raw"]
```

Comment:
I get en Exection Error: "reason_why_stopped": "Entity or transform configuration specified an unknown operation 'get-raw'. Don't know what to do." Is get-raw supposed to be added in the system config as a route as well?